renovate.json

```diff
@@ -10,6 +10,18 @@
         "matchBaseBranches": ["release-2.10","release-2.9"],
         "packagePatterns": ["*"],
         "enabled": false
+      },
+      {
+        "packagePatterns": [
+          "github.com/grafana/mimir-prometheus",
+          "github.com/grafana/memberlist",
+          "github.com/grafana/regexp",
+          "github.com/colega/go-yaml-yaml",
+          "github.com/charleskorn/goautoneg",
+          "github.com/grafana/opentracing-contrib-go-stdlib"
+        ],
+        "matchCurrentVersion": "/^v\\d+.\\d+.\\d+-(\\d+\\.)?\\d{14}-[a-f0-9]+/",
```

Comment:
Do we need `matchCurrentVersion` when we are just disabling these package patterns?